tools/flang2/flang2exe/iliutil.cpp

```diff
@@ -2093,7 +2093,7 @@ reciprocal_division(int n, INT divisor, int sgnd)
   } else {
     INT64 twol;
     int shpre = 0;
-    INT64 one_64 = {0x0, 0x1}, divisor64 = {0x0, udiv};
+    INT64 one_64 = {0x0, 0x1}, divisor64 = {0x0, (signed)udiv};
```

Comment:
Generally we are attempting to become more type safe by removing casts.  Is there a way to achieve a fix without introducing a cast?

Reply:
There is.